common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationSettings.java

```diff
@@ -88,8 +93,8 @@
      *
      * @return byte[] secret data
      */
-    public byte[] getSecretKeyData() {
-        return mSecretKeyData.get();
```

Comment:
So are we no longer using getters and setters? Do we still need them?

Reply:
we're using them.  Just adding synchronization/safe-copy.

Reply:
Ohh okay, so `Arrays.copyOf()` acts as a copy + get.....hadn't used this one before